packages/contracts/src/dollar/facets/BancorFormulaFacet.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {LibBancorFormula} from "../libraries/LibBancorFormula.sol";
+import {Modifiers} from "../libraries/LibAppStorage.sol";
+
+/**
+ * @title Bancor formula
+ * @dev Modified from the original
+ * @notice forked from Bancor:
+ * https://github.com/bancorprotocol/contracts
+ *
+ */
+contract BancorFormulaFacet is Modifiers {
+    /**
+     * @dev Given a token supply, reserve balance, weight and a deposit amount (in the reserve token),
+     * calculates the target amount for a given conversion (in the main token)
+     *
+     * @dev _supply * ((1 + _tokensDeposited / _connectorBalance) ^ (_connectorWeight / 1000000) - 1)
+     *
+     * @param _tokensDeposited   amount of collateral tokens to deposit
+     * @param _connectorWeight   connector weight, represented in ppm, 1 - 1,000,000
+     * @param _supply          current Token supply
+     * @param _connectorBalance   total connector balance
+     * 
+     * @return amount of Tokens minted
+     */
+    function _purchaseTargetAmount(
```

Comment:
we can remove the underscore in the name as it is an external func

Reply:
actually we don't need that facet has no contract will need an external call to these funcs 
so better merge the libbancorformula with libbonding curve